src/mbgl/renderer/tile_pyramid.hpp

```diff
@@ -24,7 +24,7 @@ class RenderTile;
 class RenderStyle;
 class RenderedQueryOptions;
 class SourceQueryOptions;
-class TileParameters;
+struct TileParameters;
```

Comment:
Please keep this as `class` -- it's better to just always use `class` so we don't have to switch forward declarations.